documentation/design-docs/dotnet-monitor.yaml

```diff
@@ -0,0 +1,473 @@
+swagger: "2.0"
+info:
+  description: "dotnet-monitor diagnostics agent"
+  version: "1.0.0"
+  title: "dotnet-monitor"
+host: "localhost:9999"
+basePath: "/diag/v1/"
```

Comment:
There seems to be a few accepted way to do versioning. I'm listing them here, in case others have a strong preference:

1. Base path
2. Query string
3. Custom header (e.g., `Accept-Version`: `1.0`)
4. `Accept` header (like this document proposed with logs, `Accept`: `text/plain; version=0.0.4`)

The tradeoff with query string is it's possible to have an implicit version which helps the scenario you describe.

I'm okay with query string with one difference. I'd rather the absence of a version query string default to the latest API version as opposed to version 1. Is that reasonable?

Reply:
Default = latest seems fine to me